src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -377,7 +379,7 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
         {!hasToolbar && breadcrumbs ? <ScreenreaderOnly>{breadcrumbs}</ScreenreaderOnly> : null}
         <SkeletonLayout
           style={{
-            [globalVars.stickyVerticalTopOffset]: `${verticalOffsets.header}px`,
+            [globalVars.stickyVerticalTopOffset]: `${verticalOffsets.header}px + ${resolvedStickyNotifications && notificationsHeight > 0 ? tokens.spaceXs : '0px'}`,
```

Comment:
You may ask why this space is calculated here, not in the `computeVerticalLayout` or not just calculate it as part of notifications height.

There are 2 reasons:
1. Since now notifications height includes only actual content size, not the _wrapper_ size, we need to take this space somewhere
2. It's not feasible to include this to `computeVerticalLayout`, because `tokens.spaceXs` is not a `number`, but css token (string)

If you have better suggestion how to implement it, let's discuss it in this thread